bootstrap/http_server.go

```diff
@@ -31,13 +30,13 @@ func createFiberApp() *fiber.App {
 	return fiber.New()
 }
 
-func createCityController(db *gorm.DB, q *dao.Query) *controllers.CityController {
-	repo := repositories.NewCityRepo(db, q)
+func createCityController(q *dao.Query) *controllers.CityController {
+	repo := repositories.NewCityRepo(q)
 	return controllers.NewCityController(repo)
 }
```

Comment:
To be truly FX driven, the NewCityRepo can be used as a provider directly
the city controller will then look like
```go
func createCityController(repo *repositories.CitController) *controllers.CityController {
	return controllers.NewCityController(repo)
}
```
That way, when you change the signature of NewCityRepo, you don't have to go back and change it in the bootstrap. Same for forecast